.aicoe-ci.yaml

```diff
@@ -2,12 +2,12 @@
 # Example `.aicoe-ci.yaml` with a full list of config options is available here: https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml
 check:
   # Uncomment following line to build a public image of this repo
-   - thoth-build
+  - thoth-build
 # Uncomment following lines to build a public image of this repo
 build:
```

Comment:
you miss some paramters, have a look at: https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.aicoe-ci.yaml